change/@fluentui-react-next-2020-05-26-16-20-54-xgao-theme-swtich.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "none",
+  "comment": "Storybook: add theme switching knob",
+  "packageName": "@fluentui/react-next",
```

Comment:
What about react-button?

Reply:
we can certainly do the same for `react-button`. but what theme would you load?